pages/404.js

```diff
@@ -1,14 +1,93 @@
-import styled from 'styled-components';
+import styled, { useTheme } from 'styled-components';
+import { motion } from 'framer-motion';
+import portal from '../images/portal.svg';
+import Image from 'next/image';
+import { LinkPrimaryOutline } from '../components/styles/StyledButton.styled';
+
+const MotionContainer = styled(motion.div)``;
+
+const FourZeroFourContainer = styled.div`
+	position: relative;
+	display: flex;
+	justify-content: center;
+`;
 
 const Title = styled.h1`
 	text-align: center;
 	color: var(--text1);
-	margin: 0 0 3.2rem;
+	position: absolute;
+	top: 50%;
+	left: 49.5%;
+	transform: translate(-50%, -50%);
+	z-index: 10;
+`;
+
+const FourZeroFourMain = styled(motion.main)`
+	height: 100%;
+	max-width: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	text-align: center;
+	overflow: hidden;
+	p {
+		padding: ${(props) => props.theme.spaces.md};
+	}
 `;
+
+const MotionImageContainer = styled(motion.div)`
+	position: relative;
+	max-width: 60rem;
+	width: 100%;
+	@media only screen and (max-width: ${(props) => props.theme.breakpoints.lg}) {
+		max-width: 34rem;
+	}
+	@media only screen and (max-width: ${(props) => props.theme.breakpoints.sm}) {
+		max-width: 26rem;
+	}
+`;
+
 export default function FourZeroFour() {
+	const theme = useTheme();
+
+	const svgVariants = {
+		hidden: {
+			opacity: 0,
+		},
+		visible: {
+			opacity: 1,
+			transition: {
+				duration: 0.5,
+			},
+		},
+	};
+
 	return (
-		<>
-			<Title>404</Title>
-		</>
+		<FourZeroFourMain
+			variants={theme.motion.pageTransitionVariants}
+			initial='hidden'
+			animate='visible'
+			exit='exit'
+		>
+			<MotionContainer
+				variants={svgVariants}
+				animate='visible'
```

Comment:
This line isn't required, in framer motion, as long as a motion element's parents have the 'initial' and 'animate' properties defined, and they're the same, you only need to define them on the parents.

Reply:
Please excuse my lack of experience with this library. I didn't note that in the docs either. Thanks for bringing this to my attention, it will be noted for future development.